tests/test_write_binary.py

```diff
@@ -1,7 +1,6 @@
 import numpy as np
-from tsdf import read_tsdf
-from tsdf import write_binary
-
+import pandas as pd
+from tsdf import read_tsdf, write_binary, TSDFMetadata
 
 def test_write_binary(shared_datadir):
```

Comment:
This (and the referenced implementation `write_binary.write_binary_file()`) should probably be renamed to `rite_binary.write_numpy_to_binaries`